src/device/mod.rs

```diff
@@ -1483,7 +1505,7 @@ impl Runtime {
     }
 
     async fn set_config(&mut self, config: &Option<Config>) -> Result {
-        if self.features.post_quantum_vpn.is_some() && config.is_some() {
+        if self.entities.postquantum_wg.is_running() && config.is_some() {
             // Post quantum VPN is enabled and we're trying to set up the meshnet
             return Err(Error::MeshnetUnavailableWithPQ);
         }
```

Comment:
I have some thoughts about this:
- what happens if we have regular meshnet connection already, why do we not care about that case but only pq?
- If it's necessary to have this guard, maybe moving it into the call site would be beneficial to make this function know less?

Reply:
* it should not happen since we block the PQ connection when meshnet is ON
* this function is pretty much exposed with ffi call. The check needs to happen here

Reply:
Yikes, I misunderstood the flow here, it's about meshnet and VPN, so yeah it makes sense to guard against the PQ+Meshnet=notworking. In such case is it even worth checking if `config` is Some?

Reply:
It's not necessary to check but if the caller passes `None` in here it's no harm either. The scope of things we accept is broader.